domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -1097,20 +1098,20 @@ public static void setOptimizedSqlSize(int size) {
     }
   }
 
-  private static <T, C> C collection(Class<T> clazz, Class<C> collectionClazz, ResultSet resultSet, int limit)
+  private static <T, C> C collection(Class<T> clazz, Class<C> collectionClazz, RowSet<Row> resultSet, int limit)
       throws ReflectiveOperationException, IOException {
 
-    List<JsonObject> jsonList = resultSet.getRows();
-    List<T> recordList = new ArrayList<>(jsonList.size());
     int totalRecords = 0;
-    for (JsonObject object : jsonList) {
-      String jsonb = object.getString(JSON_COLUMN);
+    List<T> recordList = new ArrayList<>(resultSet.size());
+    RowIterator<Row> iterator = resultSet.iterator();
+    while (iterator.hasNext()) {
+      Row row = iterator.next();
+      String jsonb = row.getValue(JSON_COLUMN).toString();
```

Comment:
getValue returns a JsonObject: https://github.com/eclipse-vertx/vertx-sql-client/tree/master/vertx-pg-client#supported-data-types
We don't want vertx-pg-client to create a JsonObject.
We should request a String: `SELECT jsonb::text`

Reply:
I can't that to work.

Reply:
I went through all usages of deprecated methods reported by sonar and could not find any deprecated PostgresClient.get usage:
https://sonarcloud.io/organizations/folio-org/rules?open=java%3AS1874&q=depreca#coding-rules-detail-quality-profiles

Reply:
Split off into a separate issue: https://issues.folio.org/browse/RMB-614